src/migrations/002_oauth2.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({id: 2});
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
```

Comment:
@mihok  These lines ensure that we don't re-run migrations that were run using the old system. This also means that if someone uses your 'postgres_schema/042-transfer-from-mysql.sql` file, these files also safely get skipped.